package.json

```diff
@@ -14,17 +14,17 @@
     "npm": ">=8"
   },
   "dependencies": {
-    "@actions/core": "^1.9.1",
-    "@actions/github": "^5.0.3",
+    "@actions/core": "^1.10.0",
+    "@actions/github": "^5.1.1",
     "@actions/io": "^1.1.2",
     "@actions/tool-cache": "^2.0.1"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.36.1",
+    "@typescript-eslint/eslint-plugin": "^5.38.1",
```

Comment:
`typescript-eslint/parser` should have the same version with this right? and seems like the latest version is 5.40.1

Reply:
Hmm, yes, they should, let me adjust. In terms of latest... I mostly followed the dependabot suggestions... but given it's a dev dependency, feels safe to upgrade to latest (will do with the other actions also).